src/app/components/meta/meta.component.html

```diff
@@ -136,17 +136,36 @@
     style="display: block;"
   ></app-add-tag-component>
 
-  <app-view-tags-component
-    [tags]="video | tagDisplayPipe : showManualTags
-                                   : showAutoFileTags
-                                   : showAutoFolderTags
-                                   : tagViewUpdateHack"
-    [darkMode]="darkMode"
-    (tagClicked)="filterThisTag($event)"
-    (removeTagEmit)="removeThisTag($event)"
-    class="view-tags-holder"
-    [ngStyle]="{ 'min-height': imgHeight - 128 + 'px' }"
-  ></app-view-tags-component>
+  <div *ngIf="!sortAutoTags; else sortedTags">
+    <app-view-tags-component
+   
+      [tags]="video | tagDisplayPipe : showManualTags
+                                     : showAutoFileTags
+                                     : showAutoFolderTags
+                                     : tagViewUpdateHack"
+      [darkMode]="darkMode"
+      (tagClicked)="filterThisTag($event)"
+      (removeTagEmit)="removeThisTag($event)"
+      class="view-tags-holder"
+      [ngStyle]="{ 'min-height': imgHeight - 128 + 'px' }"
+    ></app-view-tags-component>
+  </div>
+
+  <ng-template #sortedTags>
+    <app-view-tags-component
+  
+      [tags]="(video | tagDisplayPipe : showManualTags
+                                    : showAutoFileTags
+                                    : showAutoFolderTags
+                                    : tagViewUpdateHack)
+                      | autoTagSortPipe"
```

Comment:
Rather than duplicating the block and adding the pipe here, a simpler solution would be to have the pipe accept a boolean for whether to filter the contents 😉 

I tested locally and it works great, so I'll make this small change after merging 🤝